### DISCUSSION TITLE
[Help]: centered and 3 Slides per view

### DISCUSSION THREAD

**Discussion by janstieler** [davidjerleke/embla-carousel#921] [Help] [answered]
### Summary

Hi,
I have an slideshow where I use three slides per view and it is centered.
The centering works but the first slide is not centered and the last is one slide too left from center.
I don't know if I did a mistake in my css what it will make the centering wrong.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Additional information

```js
Here I have a codepen: https://codepen.io/janstieler-the-bashful/pen/PoveqNX?editors=1111
```

**Comment by sarussss** [comment, marked as answer]
Hi,
you can [check it](https://codesandbox.io/s/gsdkkq)

**Reply by janstieler** [reply]
Yes 33% makes sense if you want a three slide slideshow 😄.
